SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -743,7 +752,9 @@ int64 USpatialActorChannel::ReplicateActor()
 
 	bForceCompareProperties = false;		// Only do this once per frame when set
 
-	return (bWroteSomethingImportant) ? 1 : 0;	// TODO: return number of bits written (UNR-664)
+	INC_DWORD_STAT_BY(STAT_SpatialNumReplicatedActorBytes, ReplicationBytesWritten);
+
+	return ReplicationBytesWritten * 8;
```

Comment:
Isn't it bits if you're doing a * 8 :o

Reply:
Yeah and that's what ReplicateActor returns.

Reply:
Get in the game Josh

Reply:
My point is, shouldn't the var be called `ReplicationBitsWritten` if we then have to convert it to bytes when we return by multiplying it by 8.  Also doesn't it mean the stat `STAT_SpatialNumReplicatedActorBytes` should be bits? Sorry if I'm missing something basic here.

Reply:
How many bits are in 2 bytes:
2 (Bytes) * 8 = 16 (Bits)

Reply:
Ok my idiot streak was a success here. Yeah I follow now - brain fart.